ubuntu2204-docker/SignAndTest.Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM ubuntu:22.04 AS base_fetcher
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update && \
+    apt-get -y install --no-install-recommends \
+    ca-certificates unzip curl git jq libatomic1 \
+    && apt-get clean
```

Comment:
we might want to `rm -rf /var/lib/apt/lists/*` in addition to `apt-get clean`

Reply:
I (hopefully) added the `rm -rf` to all `apt install` locations. `apt-get clean` is apparently automatically executed in ubuntu based docker images.